pkg/controller/release/rolloutblock.go

```diff
@@ -0,0 +1,55 @@
+package release
+
+import (
+	"fmt"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/util/runtime"
+
+	shipper "github.com/bookingcom/shipper/pkg/apis/shipper/v1alpha1"
+	rolloutBlockOverride "github.com/bookingcom/shipper/pkg/util/rolloutblock"
+)
+
+func (s *Scheduler) processRolloutBlocks(rel *shipper.Release) (bool, string) {
+	relOverrideRB, ok := rel.Annotations[shipper.RolloutBlocksOverrideAnnotation]
```

Comment:
This code looks quite identical to https://github.com/bookingcom/shipper/pull/106/files#diff-3b18da970d68d60671e98b680f1b9d06R10
Do you think there is still some space for DRY'ing?